site/docs/transform/window.md

```diff
@@ -0,0 +1,131 @@
+
+## Window Field Definition
+
+{: .suppress-error}
+```json
+// A View Specification
+{
+  ...
+  "transform": [
+    {
+      // Window Transform
+      "window": [{
+          "op": "...",
+          "field": "...",
+          "as": "..."
+      }],
+      "groupby": [
+          "..."
+      ],
+      "frame": "..."
+    }
+     ...
+  ],
+  ...
+}
+```
+
+## Window Transform Definition
+
+| Property    | Type | Description  |
+| :----------- | :-------: | :------------|
+| window | WindowFieldDef[] | The definition of the fields in the window, and what calculations to use. |
+| frame | Number[] | A two element specification about how large the sliding window is. The first element indicates the start and the second element indicates the end. If `null` is specified for the start, it will include everything before the current point. If `null` is specified for the end, it will include everything after the endpoint. For example a frame of `[-5,5]` says the window should include 5 previous objects and 5 after objects. The default is `[null, 0]`, which means include everything in the window. `[null, null]` would mean include everything in the window. |
+| ignorePeers | boolean | Will indicate whether to ignore peer values (items with the same rank) in the window. The default value is `False`.|
+| groupBy | String[] | The names of the data fields to partioin the objects into seprate windows. If not specified, everything will be in a single group. |
+| sort | WindowSortField | A definition for sorting the objects within the window. Equivalent objects are considered a peer (Look at ignorePeers). If left undefined, the order of items in the window is undefined.|
+
+### Window Transform Field Definition
+| Property    | Type | Description  |
+| :----------- | :-------: | :------------|
+| field | String | The name of the field to sort.  |
+| order | `'ascending' | 'descending'` | Whether to sort the field in ascending or descending order |
+
+### Window Sort Field Definition
+| Property    | Type | Description  |
+| :----------- | :-------: | :------------|
+| op | AggregateOp or WindowOnlyOp | See supported operations in the table below.  |
+| param | number | Parameter values for the window functions. Parameter value can be null for operations that do not accept a parameter. |
+| field | string | The data field for which to compute the aggregate or window function. This can be null for functions that do not operate over a field such as `count`, `rank`, `dense_rank`. |
+| as | string | The output name for each field. If non specified will use the format `window_op_field` for example, `count_field` for count and `sum_field` for sum. |
+
+## <a name="ops"></a> Window Only Operation Reference
+
+The valid operations include all [valid aggregate operations](../aggregate/#ops) plus the following window operations.
```

Comment:
broken link  -- I think `aggregate.html#ops` is the right one, but please run `yarn run site` to check.